<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertLinkDialog</class>
 <widget class="QDialog" name="insertLinkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>165</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Insert link</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="urlLayout">
       <item>
        <widget class="QLabel" name="urlLabel">
         <property name="text">
          <string>URL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="schemeComboxBox">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string notr="true" extracomment="Link to classified ads operator profile">caprofile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true" extracomment="Link to classified ad">caad</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true" extracomment="Link to operator profile comment">cacomment</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true" extracomment="Link to binary file inside classified ads">cablob</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">http</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true" extracomment="Link to ftp resource outside classified ads">ftp</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="separatorCharsLabel">
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="statusTip">
          <string notr="true"/>
         </property>
         <property name="whatsThis">
          <string notr="true"/>
         </property>
         <property name="accessibleName">
          <string notr="true"/>
         </property>
         <property name="accessibleDescription">
          <string notr="true"/>
         </property>
         <property name="text">
          <string notr="true">://</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="urlEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="linkEditLayout">
       <item>
        <widget class="QLabel" name="displayNameLabel">
         <property name="text">
          <string>Link label in text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linkLabelEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="bottomButtonsBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bottomButtonsBox</sender>
   <signal>rejected()</signal>
   <receiver>insertLinkDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
